app/policies/user_policy.rb

```diff
@@ -80,7 +80,7 @@ def resolve
       elsif current_user.admin?
         scope.web_users.where(role: %w(admin super_organisation_admin organisation_admin normal))
       elsif current_user.super_organisation_admin?
-        scope.web_users.where(role: %w(super_organisation_admin organisation_admin normal)).where(organisation_id: current_user.organisation_id)
+        scope.web_users.where(role: %w(super_organisation_admin organisation_admin normal)).where(organisation_id: child_organisation_ids)
```

Comment:
Hyperoptimisation: if you did `where(organisation: current_user.organisation.subtree)` you'd avoid the SQL query from `child_organisation_ids`:

`User.where(organisation_id: organisation.subtree.pluck(:id))` #=> `SELECT organisations.id FROM organisations WHERE (organisations.id = 6 or organisations.ancestry like '6/%' or organisations.ancestry = '6')` and
`SELECT users.* FROM users WHERE users.organisation_id IN (2, 20, [SNIP: A BUNCH OF IDS], 1100, 6)`

vs.

`User.where(organisation: organisation.subtree)` #=> `SELECT users.* FROM users WHERE users.organisation_id IN (SELECT organisations.id FROM organisations WHERE (organisations.id = 6 or organisations.ancestry like '6/%' or organisations.ancestry = '6'))`

Rails knows how to merge ActiveRelation objects into queries without resolving them to their results first so you can avoid queries.